index.html

```diff
@@ -176,8 +193,8 @@ <h2 class="art-title">Profesional Art Printing Data</h2>
     </section>
     <section class="about-me-section">
       <div class="grid-container about">
-        <div class="grid-item about-img">
-          <h1 class="about-me">About me</h1>
+        <div class="grid-item grid-item1 about-img">
+          <h2 class="about-me">About me</h2>
```

Comment:
Kindly adjust the spacing in the about section as the previous reviewer asked.
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/159966907-fbf751e5-501e-450f-baf3-3b36418f0ecc.png) | ![image](https://user-images.githubusercontent.com/24703171/159966538-93f3b4e5-36eb-4360-a7a1-211cb16dd0df.png) |